docs/modules/ROOT/pages/hazelcast-platform-operator-map-store-mongodb-atlas.adoc

```diff
@@ -0,0 +1,275 @@
+= Configure MongoDB Atlas as an External Data Store for the Cluster with Hazelcast Platform Operator
+:page-layout: tutorial
+:page-product: operator
+:page-categories: Kubernetes
+:page-lang: java
+:page-enterprise: false
+:page-est-time: 20 mins
+:description: Learn how to keep data in sync across two Hazelcast clusters.
+
+== Context
+In this tutorial, you'll do the following:
+
+- Build and deploy a custom MapStore/MapLoader implementation to cloud storage.
+
+- Start Hazelcast cluster with classpath that includes MapStore/MapLoader JAR
+
+- Create a Hazelcast Map with MapStore/MapLoader configuration
+
+- Run Hazelcast Java client to manipulate data via populating all entries from MongoDB Atlas.
+
+
+== Before you Begin
+
+Before starting this tutorial, make sure that you meet the following prerequisites:
+
+* Up and running *public* https://kubernetes.io/[Kubernetes] cluster. Make sure the Kubernetes cluster is public.
+* Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
+* Maven command-line tool, https://maven.apache.org/download.cgi[mvn]
+* MongoDB Atlas free-tier database, https://www.mongodb.com/docs/atlas/getting-started[Get Started with Atlas]
```

Comment:
```suggestion
* MongoDB Atlas database, https://www.mongodb.com/docs/atlas/getting-started[Get Started with Atlas] (free-tire is suitable for completing this guide).
```
Maybe the wording can be different, but my point is that the prerequisite is MongoDB Atlas DB, not exactly free-tier DB 😃